### DISCUSSION TITLE
Show urls ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [tadashi-aikawa/obsidian-another-quick-switcher#253] [Q&A] [answered]
Hello,

There is `link search` that list local links but is there a way to the same for the urls used in the current note so we can open them from a modal ?

**Comment by tadashi-aikawa** [comment, marked as answer]
Hi, @Fred-Vatin :)

> There is link search that list local links but is there a way to the same for the urls used in the current note so we can open them from a modal ?

No, there isn't a method like that at the moment.

**Reply by Fred-Vatin** [reply]
The repo only accept bug as template in issue. Where can I post this as a feature request ?

**Reply by tadashi-aikawa** [reply]
You can create from here. For non-bug reports, the format is free 👍 

![image](https://github.com/user-attachments/assets/7d3adbe9-abd7-47d8-93c2-6b8fa93c34a1)
